test/core/event_engine/event_engine_test_utils.cc

```diff
@@ -241,5 +217,29 @@ ConnectionManager::CreateConnection(std::string target_addr,
   return absl::CancelledError("Failed to create connection.");
 }
 
+// Returns a random message with bounded length.
+std::string GetNextSendMessage() {
```

Comment:
Any specific reason why this function was moved to the end of this file from the beginning of the file ?

Reply:
No, it was a merge artifact, we had both done similar changes here. I've moved it back up near the top.